src/main/java/com/google/codeu/data/Place.java

```diff
@@ -47,6 +57,65 @@ public Place(UUID id, String owner, String title, String description, long latit
     this.timestamp = timestamp;
   }
 
+  /** Return Place data using based on entity from search query. */
+  public Place(Entity entity) {
+    this.id = (UUID)entity.getProperty("id");
+    this.owner = (String)entity.getProperty("owner");
+    this.title = (String)entity.getProperty("title");
+    this.description = (String)entity.getProperty("description");
+    this.latitude = (long)entity.getProperty("latitude");
+    this.longitude = (long)entity.getProperty("longitude");
+    this.timestamp = (long)entity.getProperty("timestamp");
+  }
+
+
+  /** Stores a new Place in Datastore. */
+  public void storePlace(Place place) {
```

Comment:
These should be static, and they should have different names.

store()
getAll()
getByUser()